packages/rollups-node/s6-overlay/s6-rc.d/sunodo-node/up

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+sunodo-node --port 3000
```

Comment:
To use unprivileged `cartesi` user.

The timeout configuration could be different depending on how much time it takes to start this node version.

```suggestion
#!/command/with-contenv sh
exec s6-notifyoncheck -w 500 -t 500 -n 5 \
    s6-setuidgid cartesi sunodo-node --port 3000
```